modules/tinymce/src/core/main/ts/selection/GetSelectionContentImpl.ts

```diff
@@ -90,10 +85,12 @@ const setupArgs = (args: Partial<GetSelectionContentArgs>, format: ContentFormat
   ...args,
   format,
   get: true,
-  selection: true
+  selection: true,
+  content: '',
```

Comment:
This should have been reverted since we found out the existing type is meant to be empty/optional in this case.
```suggestion
```